hatch.toml

```diff
@@ -62,6 +62,7 @@ dependencies = [
   "pymdown-extensions~=10.8.1",
   # Necessary for syntax highlighting in code blocks
   "pygments~=2.18.0",
+  "griffe<1.0",
```

Comment:
(how) is this related to the proposed prose changes?

Reply:
hey @funkyfuture i mentioned this to @ofek right now the docs won't build without this change in the hatch doc environment. I read it's a known issue. I think ofek wants to make the change separately / rebuild the environment. But if anyone wants to pull this down and build they will have issues without this change (or atleast that is what i discovered).